book/content/documentation/documenation.md

```diff
@@ -0,0 +1,50 @@
+# Documentation
+
+## About
+Both the project and the source code should be documentated in order for others to understand what the project is about, how they can use it and how they can contribute to it.
+
+### Markdown
+[Markdown](https://en.wikipedia.org/wiki/Markdown) is a lightweight markup language with plain-text-formatting syntax. Its design allows it to be converted to many output formats, but the original tool by the same name only supports HTML. Markdown is often used to format readme files, for writing messages in online discussion forums, and to create rich text using a plain text editor.
+
+GitHub and other project and code hosting services automatically render markdown files for humans to read. 
+
+You can learn about markdown syntax and how to master this language in an excellent GitHug [Mastering markdown Guide](https://guides.github.com/features/mastering-markdown/). Try it out with an [online editor](https://markdown-it.github.io)
```

Comment:
Markdown syntax is a bit ambiguous, better to use [CommonMark](https://commonmark.org/). Can we replace this section with CommonMark sites like https://commonmark.org/help/ and  https://spec.commonmark.org/dingus/